### ISSUE TITLE
Table Color Contrast Accessibility

### ISSUE THREAD

**Issue by lfarrell** [samvera/hyrax#2168]
### Descriptive summary

Accessibility contrast for stripped tables generate WAVE errors for darker striped rows.

### Rationale

This is an accessibility issue for low vision users.

### Expected behavior

Color contrast passes WAVE validation.

### Actual behavior

![table-contrast](https://user-images.githubusercontent.com/436691/32737664-ac62368c-c868-11e7-8c63-3d012466c4d9.png)

**Comment by h-parekh** [comment]
@mtribone I think this should be fixed by https://github.com/samvera/hyrax/pull/2205

**Comment by h-parekh** [comment]
Confirmed with @mtribone, this contrast issue has been fixed in master.
